modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/Text/Text.es.js

```diff
@@ -196,10 +238,58 @@ describe('Field Text', () => {
 
 		render(
 			<div className="ddm-page-container-layout">
-				<TextWithProvider {...props} />
+				<TextWithProvider {...props} key="input" onChange={onChange} />
 			</div>
 		);
 
+		const input = document.body.querySelector('input');
+
+		fireEvent.change(input, {
+			target: {
+				value: '',
+			},
+		});
+
+		act(() => {
+			jest.runAllTimers();
+		});
+
+		const autocompleteDropdownMenu = document.body.querySelector(
+			'.autocomplete-dropdown-menu'
+		);
+
+		const classList = autocompleteDropdownMenu.classList;
+
+		expect(classList.contains('show')).toBeFalsy();
```

Comment:
This test checks if the auto-complete drop-down is hidden when the input field is empty, so it really expects `contains('show')` to be falsy because of that.